ls1intum/Artemis
# `Exam mode`: Show exam sessions

### Checklist
#### General


#### Server
#### Client


### Motivation and Context
* Show the number of exam sessions per student in the student exam page
* Show all exam session details in the student exam detail view, in case exam sessions are available

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Exam

1. Participate in the exam and reload the page several times in different browsers, potentially also from different devices using different IP addresses  
2. Navigate to the student exam page and check if for each time you start the exam (i.e. entering your name in the start exam screen), there is a new exam session
3. Inspect the details of the exam sessions in the student exam detail page


### Review Progress
#### Code Review
#### Manual Tests

### Screenshots
<img width="1503" alt="image" src="https://user-images.githubusercontent.com/744067/219970181-6274dbf3-f9fa-4359-9683-949f1a12c193.png">
<img width="1498" alt="image" src="https://user-images.githubusercontent.com/744067/219970200-db302e50-6788-4dec-a186-bf447261c1ba.png">
<img width="1495" alt="image" src="https://user-images.githubusercontent.com/744067/219970282-ec60c47f-3965-4868-8a57-afd4a078f1ab.png">
